.github/workflows/manual-deploy-ten-gateway-frontend.yml

```diff
@@ -15,27 +15,16 @@ on:
           - 'dev-testnet'
           - 'uat-testnet'
           - 'sepolia-testnet'
-
 jobs:
   build-and-deploy:
     runs-on: ubuntu-latest
     environment:
       name: ${{ github.event.inputs.testnet_type }}
-    strategy:
-      fail-fast: false
-      matrix:
-        include:
-          - testnet_type: 'dev-testnet'
-            GATEWAY_API_URL: 'https://dev-testnet.ten.xyz'
-          - testnet_type: 'uat-testnet'
-            GATEWAY_API_URL: 'https://uat-testnet.ten.xyz'
-          - testnet_type: 'sepolia-testnet'
-            GATEWAY_API_URL: 'https://testnet.ten.xyz'
     steps:
       - name: 'Print GitHub variables'
         run: |
           echo "Selected Testnet Type: ${{ matrix.testnet_type }}"
-          echo "Gateway API URL: ${{ matrix.GATEWAY_API_URL }}"
+          echo "Gateway API URL: ${{ vars.GATEWAY_URL }}"
```

Comment:
how is this differentiated for each env?

Reply:
we have separate environments (https://github.com/ten-protocol/go-ten/settings/environments ) and for each environment we need `GATEWAY_URL` variable which represents the URL for that specific testnet type.